tests/testthat/test-les-kb-v2.R

```diff
@@ -1,163 +1,163 @@
+)
+
+# kb_eksempel er de fire første linjene fra AblaNor kodebok.
+# Den inkluderer de kolonnene vi ønsker ut, og riktig kolonnetype for alle kolonner.
+kb_eksempel = data.frame(
+  stringsAsFactors = FALSE,
+  skjemanavn = c("Personopplysninger", "Personopplysninger", "Personopplysninger", "Basisskjema"),
+  navn_i_rapporteket = c(NA_character_, NA_character_, NA_character_, NA_character_),
+  ledetekst = c("PID", "Dato for innhenting av opplysninger", "Fødselsnummer", "Årstall for debut arytmi"),
+  obligatorisk = c("Ja", "Ja", "Ja", "Nei"),
+  type = c("Tekstvariabel", "Datovariabel", "Tekstvariabel", "Tallvariabel"),
+  listeverdier = c(NA_character_, NA_character_, NA_character_, NA_character_),
+  listetekst = c(NA_character_, NA_character_, NA_character_, NA_character_),
+  normalintervall_start_numerisk = c(NA_real_, NA_real_, NA_real_, "birthYear"),
+  normalintervall_slutt_numerisk = c(NA_real_, NA_real_, NA_real_, "todayYear"),
+  maksintervall_start_numerisk = c(NA_real_, NA_real_, NA_real_, "birthYear"),
+  maksintervall_slutt_numerisk = c(NA_real_, NA_real_, NA_real_, "todayYear"),
+  normalintervall_start_dato = c(NA_character_, NA_character_, NA_character_, NA_character_),
+  normalintervall_slutt_dato = c(NA_character_, NA_character_, NA_character_, NA_character_),
+  maksintervall_start_dato = c(NA_character_, "'1980-01-01'", NA_character_, NA_character_),
+  maksintervall_slutt_dato = c(NA_character_, "today", NA_character_, NA_character_),
+  antall_tegn = c(11L, NA_integer_, 11L, 4L),
+  lovlige_tegn = c(
+    "1234567890", NA_character_,
+    "aAáÁbBcCdDðÐeEéÉfFgGhHiIíÍjJkKlLmMnNoOóÓpPqQrRsStTuUvúÚVwWxXyYýÝzZþÞæÆøØåÅäÄöÖñÑéÉ-ü ÜáÁ´_/,.- + 1234567890ØøÜüß@_-!%:;?",
+    NA_character_
+  ),
+  desimaler = c(NA_integer_, NA_integer_, NA_integer_, 0L),
+  aktiveringsspoersmaal = c("Nei", "Nei", "Nei", "Nei"),
+  underspoersmaal = c("Nei", "Nei", "Nei", "Nei"),
+  innfoert_dato = c("1980-01-01", "1980-01-01", "1980-01-01", "2017-04-25"),
+  utfaset_dato = c(NA_character_, NA_character_, NA_character_, NA_character_),
+  tabell = c("patient", "patient", "patient", "basereg"),
+  fysisk_feltnavn = c("ID", "REGISTERED_DATE", "SSN", "DEBUT_ARYT_AAR"),
+  kommentar = c(NA_character_, NA_character_, NA_character_, NA_character_),
+  variabel_id = c("PATIENT_ID", "PATIENT_REGISTERED_DATE", "PATIENT_SSN", "BASEREG_DEBUT_ARYT_AAR"),
+  hjelpetekst = c(
+    "Pasient ID - automatisk løpenummer i databasen.",
+    "Skriv inn dato for innhentings tidspunkt for opplysninger. Dato skrives på formatet yyyymmdd. Skriver du inn kun dag (dd), så autfylles nåværende måned og år. Skriver du inn måned og dato (mmdd), så autofylles nåværende år. Eksempel: For dato 23.11.1980 er følgende verdier gyldig: 801123, eller 19801123, eller 1980-11-23", "Skriv inn fødselsnummer, 11 siffer. Følgende format for fødselsnummer: ddmmyyxxxxx", "Angi årstall for debut arytmi"
   )
-  kb_eksempel = as_tibble(kb_eksempel)
-}
+)
+kb_eksempel = as_tibble(kb_eksempel)
+
```

Comment:
Korfor vert objektet først laga som ein `data.frame()` (med `stringsAsFactors = FALSE`) for så etterpå bli gjort om til ein `tibble()`?

Reply:
Heilt unødvendig. Gjort om.